src/chat/infra/repo/MongoChatRepository.ts

```
@@ -28,11 +27,12 @@ type ChatUpdateFields = {
 };
 
 export class MongoChatRepository implements IChatRepository {
-  // Constants
   private readonly BOT_ID = CONFIG.BOT_ID;
   private readonly ADMIN_ID = CONFIG.ADMIN;
 
-  // Helper Methods
+  constructor() {
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)